filament/src/details/Engine.cpp

```diff
@@ -167,7 +196,7 @@ static constexpr float4 sFullScreenTriangleVertices[3] = {
 // these must be static because only a pointer is copied to the render stream
 static const uint16_t sFullScreenTriangleIndices[3] = { 0, 1, 2 };
 
-FEngine::FEngine(Backend backend, Platform* platform, void* sharedGLContext) :
+FEngine::FEngine(Backend backend, Platform* platform, void* sharedGLContext, const Config& configParams) :
```

Comment:
I think it's fine to use a reference here (so it can't be nullptr) but in that case, I would move this parameter in 2nd position after `backend`.

Also name the parameters the same in the header and in the cpp (e.g. `config` in both cases).

Reply:
Fixed in my next update.